frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/obc-test.spec.ts

```diff
@@ -0,0 +1,140 @@
+import { expect } from 'chai';
+import {
+  ACCESS_KEY,
+  BOUND,
+  MASK,
+  MINUTE,
+  NS,
+  OBC_NAME,
+  OBC_STORAGE_CLASS,
+  SECOND,
+  SECRET_KEY,
+} from '../utils/consts';
+import { testDeployment, deployment } from '../mocks/deploymentData';
+import { CreateOBCHandler } from '../views/obcPage';
+import { testName } from '../../../integration-tests-cypress/support/index';
+
+describe('Test Object Bucket Claim resource', () => {
+  let obcHandler;
+  let obcUrl;
+
+  before(() => {
+    cy.url().then((url) => {
+      if (url.match(/oauth/)) {
```

Comment:
I am not really a fan of this approach. As conditional flows reflects that we don't know the actual state of the application, which violates Cypress principles. PTAL https://docs.cypress.io/guides/core-concepts/conditional-testing.html#Definition

Reply:
+1. This is something I did to "workaround" the flakiness of the `cy.login` method, but I'm not a fan as well if it introduces anti-patterns in the tests. I'll replace this with something that simply checks for this case and fails if the condition is not met, instead of incorporating a workaround.

```
    cy.url().should((url) => {
      expect(url).to.match(/oauth/);
    });
    cy.login();
```